applications/GeoMechanicsApplication/tests/cpp_tests/test_stress_strain_utitlities.cpp

```diff
@@ -0,0 +1,120 @@
+{
+    const Vector eye = MathUtils<double>::StressTensorToVector(IdentityMatrix(3));
+    KRATOS_EXPECT_DOUBLE_EQ(3.0, StressStrainUtilities::CalculateTrace(eye));
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateMeanStress, KratosGeoMechanicsFastSuite)
+{
+    const Vector eye = MathUtils<double>::StressTensorToVector(IdentityMatrix(3));
+    KRATOS_EXPECT_DOUBLE_EQ(1.0, StressStrainUtilities::CalculateMeanStress(eye));
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckGreenLagrangeStrainTensor, KratosGeoMechanicsFastSuite)
+{
+    const Matrix eye =
+        StressStrainUtilities::CalculateGreenLagrangeStrainTensor(std::sqrt(3.) * IdentityMatrix(3));
+    KRATOS_EXPECT_MATRIX_NEAR(eye, IdentityMatrix(3), 1e-12)
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateVonMisesStressHydrostatic, KratosGeoMechanicsFastSuite)
+{
+    Vector stress_vector(4);
+    stress_vector <<= -2.0, -2.0, -2.0, 0.0;
+    KRATOS_EXPECT_DOUBLE_EQ(0.0, StressStrainUtilities::CalculateVonMisesStress(stress_vector));
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateVonMisesStressPureShear, KratosGeoMechanicsFastSuite)
+{
+    Vector stress_vector(4);
+    stress_vector <<= 0.0, 0.0, 0.0, 2.0;
+    KRATOS_EXPECT_DOUBLE_EQ(2.*std::sqrt(3.), StressStrainUtilities::CalculateVonMisesStress(stress_vector));
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateLodeAngle, KratosGeoMechanicsFastSuite)
+{
+    Vector stress_vector(4);
+    stress_vector <<= 0.5, -1.0, 0.5, 0.0;
+    KRATOS_EXPECT_DOUBLE_EQ(MathUtils<>::DegreesToRadians(30.),
+                            StressStrainUtilities::CalculateLodeAngle(stress_vector));
+    KRATOS_EXPECT_DOUBLE_EQ(MathUtils<>::DegreesToRadians(-30.),
+                            StressStrainUtilities::CalculateLodeAngle(-1. * stress_vector));
+    Vector stress_vector2(4);
+    stress_vector2 <<= -1.0, 0.0, 1.0, 0.0;
+    KRATOS_EXPECT_DOUBLE_EQ(MathUtils<>::DegreesToRadians(0.),
+                            StressStrainUtilities::CalculateLodeAngle(stress_vector2));
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateMohrCoulombShearCapacityZeroStress, KratosGeoMechanicsFastSuite)
+{
+    const Vector stress_vector  = ZeroVector(4);
+    const double cohesion       = 2.;
+    const double friction_angle = 0.;
+    KRATOS_EXPECT_DOUBLE_EQ(0.0, StressStrainUtilities::CalculateMohrCoulombShearCapacity(
+                                     stress_vector, cohesion, friction_angle));
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateMohrCoulombShearCapacityHydrostatic, KratosGeoMechanicsFastSuite)
+{
+    Vector stress_vector(4);
+    stress_vector <<= -2.0, -2.0, -2.0, 0.0;
+    const double cohesion       = 0.;
+    const double friction_angle = MathUtils<>::DegreesToRadians(90.);
+    KRATOS_EXPECT_DOUBLE_EQ(0.0, StressStrainUtilities::CalculateMohrCoulombShearCapacity(
+                                     stress_vector, cohesion, friction_angle));
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateMCShearCapacityShearOnly, KratosGeoMechanicsFastSuite)
```

Comment:
Perhaps it is more readable and consistent to use `MohrCoulomb` rather than `MC`? If yes, then the same comment applies to the next two test cases.